spec/core_spec.rb

```diff
@@ -129,6 +129,15 @@ class Level0Serializer0 < BaseTestSerializer
       it "renders correct results" do
         expect(relationship_level1_ids).to match_array(level1_records.map(&:id))
       end
+
+      context "association populated by accepts_nested_attributes_for" do
```

Comment:
It could be unit tested as well in `loaders/association_spec.rb`. Not super necessary though.